appxmanifest.xml.in

```diff
@@ -94,6 +94,12 @@
             </uap:SupportedFileTypes>
           </uap:FileTypeAssociation>
         </uap:Extension>
+        <uap:Extension Category="windows.protocol">
+          <uap:Protocol Name="%(protocol)s">
+            <uap:Logo>Square44x44Logo.targetsize-44_altform-unplated.png</uap:Logo>
+            <uap:DisplayName>%(display-name)s</uap:DisplayName>
```

Comment:
This DisplayName is for the protocol or the app? I mean shouldn't it be %(display-name)s protocol or something like that?

Reply:
For the installer version this is what Edge displays when opening such a link:

![grafik](https://github.com/user-attachments/assets/0a1c6522-713a-415a-ac29-2b502bdb8cd9)

No icon in this case, and the name shown is taken from the description inside the `picard.exe` file (which we set at https://github.com/metabrainz/picard/blob/master/win-version-info.txt.in#L39).

I have yet to test the store version (for which the above configuration is).